src/config/server.rs

```diff
@@ -186,6 +190,9 @@ impl Server {
                 .unwrap_or_default()
         );
 
+        let domain_name = dotenvy::var("DOMAIN_NAME").unwrap_or_else(|_| "crates.io".into());
+        let trustpub_audience = var("TRUSTPUB_AUDIENCE")?.unwrap_or_else(|| domain_name.clone());
```

Comment:
this allows us to configure the expected `aud` claim, if necessary. it defaults to the domain name (crates.io / staging.crates.io).

Reply:
Nice! This may or may not be applicable in your case, but noting: PyPI intentionally chose _not_ to use the domain here, since we thought we might want to change the authentication and/or exchange subdomain at some point. So PyPI uses `pypi` and TestPyPI uses `testpypi`.

(This _probably_ doesn't matter in practice, just flagging as a very small difference! In practice, any identifier that serves as a domain separator is fine, DNS name or whatever else 🙂)